woocommerce/woocommerce
# [Accessibility] Fix focus order on checkout block page

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #43638 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Before pulling this PR to your local environment for testing, make sure your store already has the Checkout page built with the Checkout Block. The goal is to confirm that stores won't have any regressions once updates the plugin.
2. Tab navigate through the Checkout page and verify the fields on the Checkout Summary block are focused after the elements on the Checkout Form block. Repeat this process on both mobile and desktop.
3. On the admin go to Appearance -> Editor -> Templates -> WooCommerce -> Page: Checkout and verify if there are changes on the Checkout template. You can do that by clicking on the three dots button on the right side. If the Reset option is visible, it means that there are changes done using the template editor.
![image](https://github.com/user-attachments/assets/248275d4-b581-4d06-a725-68d8a2303e1c)
4. Reset the changes.
4. Pull these changes to your store and confirm the Checkout page keeps working as it was before.
6. Go to the Checkout page on the editor.
7. Delete the Checkout Block.
8. Add the Checkout Block again.
9.  Publish the page.
10. Repeat the process of tab navigating through the Checkout page and verify that now the elements on the Checkout Summary block are focused before the elements on the Checkout Form block.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
